spark/init.sh

```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 
 pushd /root > /dev/null
-
-if [ -d "spark" ]; then
-  echo "Spark seems to be installed. Exiting."
-  return
-fi
-
+rm -rf spark
+SPARK_VERSION="2.3.3"
```

Comment:
Why is SPARK_VERSION needed? Shouldn't this be passed in?

Reply:
I was debugging it, and will remove it